src/components/HistoryDialog/index.tsx

```diff
@@ -51,149 +49,236 @@ const StyledTitle = styled("p")({
 const StyledDialogContent = styled(DialogContent)({
```

Comment:
Vertical spacing between title and content, as well as content and Close button seems smaller than then UI design. As well as under the Close button.

Current:
![Screenshot 2024-10-23 at 11 35 48 AM](https://github.com/user-attachments/assets/aea9b35f-f2eb-4a6d-959f-8f77490fa383)

UI Design:
![Screenshot 2024-10-23 at 11 35 42 AM](https://github.com/user-attachments/assets/2d494185-0e17-4ab5-afa3-682c9110a721)

Reply:
Adjusted accordingly in the commit 5a27dad. I think the spacing is still slightly off, but it's an improvement. Please feel free to unresolve this if you want it adjusted more.